src/options.ts

```diff
@@ -214,16 +214,16 @@ export function getCommonCommandOpts() {
 
   const configOpt = createOption(
     '-c, --config <path>',
-    'configuration path (default: synthetics.config.js)'
+    'path to the configuration file (default: synthetics.config.(js|ts))'
   );
 
   const tags = createOption(
     '--tags <name...>',
-    'run/push tests with a tag that matches the glob'
+    'run/push tests with the tag(s) that matches a pattern'
```

Comment:
```suggestion
    'run/push tests with the tag(s) matching a pattern'
```

IMO this reads better because it is correct for `tag|tags`.